src/clj_irods/core.clj

```diff
@@ -188,30 +199,54 @@
   (otel/with-span [s ["file-size"]]
     (from-stat-or-listing :file-size :data_size irods user zone path)))
 
+(defn object-avu
```

Comment:
This function isn't actually used by data-info because I added a separate info-type one later to allow fetching that from the listing (which doesn't have the unit, but is fine when the unit isn't needed). However, it's a good thing we'll probably want, so I left it implemented.